src/tests/ftest/util/dfuse_utils.py

```diff
@@ -391,13 +399,136 @@ def get_stats(self):
+            pool.pool.handle.value if hasattr(pool.pool.handle, 'value') else pool.pool.handle)
+        sub_cont.container = DaosContainer(pool.context)
+        sub_cont.container.poh = pool.pool.handle
+
+        sub_cont.daos.json.update(True)
+        kwargs = {
+            "pool": pool.identifier,
+            "sys_name": pool.name.value,
+            "path": sub_cont.path.value,
+            "type": sub_cont.type.value,
+            "oclass": sub_cont.oclass.value,
+            "dir_oclass": sub_cont.dir_oclass.value,
+            "file_oclass": sub_cont.file_oclass.value,
+            "chunk_size": sub_cont.chunk_size.value,
+            "properties": sub_cont.properties.value,
+            "acl_file": sub_cont.acl_file.value,
+            "label": sub_cont.label.value,
+            "attrs": sub_cont.attrs.value
+        }
+        sub_cont.daos.set_command(("container", "create"), **kwargs)
+        result = run_remote(self.log, self._running_hosts[0], str(sub_cont.daos))
+        if not result.passed:
+            raise CommandFailure(
+                f"DFuse sub-container can not be created on {result.failed_hosts}: "
+                f"{result.output[0].stderr}")
+        result_json = json.loads(os.linesep.join(result.output[0].stdout))
+        try:
+            sub_cont.uuid = result_json["response"]["container_uuid"]
+        except KeyError as error:
+            raise CommandFailure("Error: Unexpected daos container create output") from error
+        sub_cont.label.update(result_json["response"].get("container_label"))
+        sub_cont.type.update(result_json["response"].get("container_type"))
+
+        sub_cont.container.uuid = str_to_c_uuid(sub_cont.uuid)
+        sub_cont.container.attached = 1
+
+        self.sub_conts.append(sub_cont)
+
+        return sub_cont
+
+    def destroy_sub_cont(self, cont):
+        """Destroy a given DFuse sub-container
+
+        Args:
+            cont (TestContainer): DFuse sub-container to destroy
+
+        Returns:
+            bool: True if the container has been destroyed; False otherwise
+
+        """
+        if not self.check_running(False):
+            self.log.info("DFuse must be running to destroy a sub-container")
+            return False
+
+        if cont not in self.sub_conts:
+            self.log.info("DAOS container %s is not a DFuse sub-container", str(cont))
+            return False
+
+        host = self._running_hosts[0]
+        kwargs = {"path": cont.path.value}
+        cont.daos.set_command(("container", "destroy"), **kwargs)
+        self.log.info("Destroying DFuse sub-container %s on %s", str(cont), host)
+        result = run_remote(self.log, host, str(cont.daos))
+        if not result.passed:
+            self.log.info(
+                "DFuse sub-container %s can not be destroyed on %s: %s",
+                cont, result.failed_hosts, result.output[0].stderr)
+            return False
+
+        self.sub_conts.remove(cont)
+        return True
```

Comment:
From my understanding, destroying sub-container should be done using the path option and not its UID.
As for the sub_create, this should be done on one of the client node which have this DFuse mount point mounted and not from the launcher node.

Reply:
If the container destroy needs to be run with the path argument - which doesn't seem very user friendly to me - then we could support that through `remove_container()` and `TestContainer.destroy` - by supporting a `**params` argument to both methods and adding the required path to the register cleanup call in `add_container()`.

Reply:
We have an existing test - dfuse/root_container.py - that creates sub containers and it does not doing anything special - other than defining a path for create - when creating and deleting the sub containers.  Is there any reason this new test is different?

Reply:
We could keep things consistent between src/tests/ftest/dfuse/root_container.py and src/tests/ftest/dfuse/container_attrs.py by removing the `test_clients: 1` line from src/tests/ftest/dfuse/container_attrs.yaml.  This would result in all the dfuse and daos commands running on the same host - the launch node for now.  

I do have a ticket to remove running these commands on the launch node and I will handle supporting running the daos command for the container create/delete via run_remote() as part of those changes.  My plan is to support that within the TestContainer class.

Reply:
Make sense to keep things consistent.
However, I am not sure if the cleanup action should be handle in the TestContainer class or in the umount of the DFuse class.  From my understanding, the first option would make sense if the container is not strictly bounded to the DFuse mount point (i.e. can not be reused for different purpose).  If not the second option seems to me more suited to me. @ashleypittman as DFuse expert, do you have an opinion on this point.

BTW, I will update this functional test in the same way as the `root_container.py`.  Could you also give me the reference of your ticket for running create/delete remotely.  I would like to add some comments to not forget to update the test when such features will be available.

- [ ] Create and destroy sub container on the launcher node

Reply:
> Make sense to keep things consistent. However, I am not sure if the cleanup action should be handle in the TestContainer class or in the umount of the DFuse class. From my understanding, the first option would make sense if the container is not strictly bounded to the DFuse mount point (i.e. can not be reused for different purpose). If not the second option seems to me more suited to me. @ashleypittman as DFuse expert, do you have an opinion on this point.
> 
> BTW, I will update this functional test in the same way as the `root_container.py`. Could you also give me the reference of your ticket for running create/delete remotely. I would like to add some comments to not forget to update the test when such features will be available.
> 
> * [x]  Create and destroy sub container on the launcher node

Fixed with commit 4e323bd67